plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptionsGroupedProductSelectorItemTemplate.php

```diff
@@ -0,0 +1,82 @@
+<?php
+declare(strict_types=1);
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+use Automattic\WooCommerce\Admin\Features\Features;
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+use WP_Block;
+
+/**
+ * Block type for grouped product selector in add to cart with options.
+ */
+class AddToCartWithOptionsGroupedProductSelectorItemTemplate extends AbstractBlock {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'add-to-cart-with-options-grouped-product-selector-product-item-template';
```

Comment:
Personal opinion, but could we make this block's name a bit shorter?

```suggestion
	protected $block_name = 'add-to-cart-with-options-grouped-product-selector-item';
```

Reply:
done